doc/getting-started.md

```diff
@@ -172,3 +180,43 @@ The default font for material design is [`Roboto`](https://fonts.google.com/spec
 	<MaterialFonts xmlns="using:Uno.Material"
 				   OverrideSource="ms-appx:///Style/Application/MaterialFontsOverride.xaml" />
 	```
+
+## Uno.Cupertino
+Uno Cupertino is an add-on package that lets you apply [Cupertino - Human Interface Guideline styling](https://developer.apple.com/design/human-interface-guidelines) to your application with a few lines of code. 
+
+It features:
+- Color system for both Light and Dark themes
+- Styles for existing WinUI controls like Buttons, TextBox, etc.
+
+### Getting Started
+1. Open an existing Uno project, or create a new Uno project using the `Uno Platform App` template.
+2. In the Solution Explorer panel, right-click on your solution name and select `Manage NuGet Packages for Solution ...`. Choose [`Uno.Cupertino`](https://www.nuget.org/packages/Uno.Cupertino/) package for Uno UWP solution or Choose the [`Uno.Cupertino.WinUI`](https://www.nuget.org/packages/Uno.Cupertino.WinUI) package for Uno WinUI solution. And, select the following projects to include it:
+	- `PROJECT_NAME.Wasm.csproj`
+	- `PROJECT_NAME.Mobile.csproj` (or `PROJECT_NAME.iOS.csproj`, `PROJECT_NAME.Droid.csproj`, `PROJECT_NAME.macOS.csproj` if you have an existing project)
+	- `PROJECT_NAME.Skia.Gtk.csproj`
+	- `PROJECT_NAME.Skia.WPF.csproj`
+	- `PROJECT_NAME.Windows.csproj` (or `PROJECT_NAME.UWP.csproj` for existing projects)
+1. Add the following resources inside `App.xaml`:
+    ```xml
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <!-- Load WinUI resources -->
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+
+                <CupertinoColors xmlns="using:Uno.Cupertino"  />
+				<CupertinoResources xmlns="using:Uno.Cupertino" />
+
+                <!-- Load custom application resources -->
+				<!-- ... -->
+
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+    ```
+
+For complete instructions on using Uno Cupertino in your projects, check out this walkthrough: [How to use Uno.Cupertino](https://platform.uno/docs/articles/guides/uno-cupertino-walkthrough.html).
```

Comment:
Isn't this going into this repo?

Reply:
@jeromelaban we will do a better clean-up after to bring the rest of Cupertino docs from uno to themes, but we will need to verify the content first and edit as needed. Right now the main goal was Material and we will go back to it after.